packages/web/src/components/header/desktop/Header.module.css

```diff
@@ -124,3 +124,14 @@
   left: 0px;
   z-index: 11;
 }
+
+.gutter {
+  right: 0px;
+  position: fixed;
+  box-shadow: rgba(133, 129, 153, 0.18) 1px 2px 5px -2px;
+  z-index: -1;
+}
+
+.bannerMargin {
+  margin-top: 40px;
```

Comment:
this stinks 😢 maybe someday we can make the banner push the container down so we don't have to do this

Reply:
At the very least, I would say make the height a CSS variable that's used both in the banner component and here?

Reply:
Didn't want to do a deeper refactor than I already did here (since this is how it worked before), but yeah agreed. I'll pull it out into a CSS variable at least